addons/mail/views/mail_activity_views.xml

```diff
@@ -194,6 +194,25 @@
         </field>
     </record>
 
+    <record id="mail_activity_view_form_minimal" model="ir.ui.view">
+        <field name="name">mail.activity.view.tree.minimal</field>
```

Comment:
```suggestion
        <field name="name">mail.activity.view.form.minimal</field>
```

Reply:
Thanks, actually, I have renamed them: mail_activity_view_form_without_record_access and mail_activity_view_tree_without_record_access so it is easier to know their purpose.